docs/src/main/sphinx/admin/preview-web-interface.md

```diff
@@ -0,0 +1,29 @@
+# Preview Web UI
+
+In addition to the [](/admin/web-interface), Trino includes a preview version of
+a new web interface. It changes look and feel, available features, and many
+other aspects. In the future this new user interface will replace the existing
+user interface.
+
+:::{warning}
+The Preview Web UI is not suitable for production usage, and only available for
+testing and evaluation purposes. Feedback and assistance with development is
+encouraged.
+:::
+
+## Activation
+
+The Preview Web UI is not available by default, and must be activated in
```

Comment:
s/activated/enabled

The Preview Web UI is not exposed?

Reply:
exposed sounds weird .. also style says not to use "disabled" ... 

but I did switch to "enabled"